src/main/java/uk/gov/hmcts/ccd/domain/model/aggregated/CaseViewFieldBuilder.java

```diff
@@ -53,4 +63,23 @@ public CaseViewField build(CaseField caseField, CaseEventField eventField, Objec
             .map(eventField -> build(caseFieldMap.get(eventField.getCaseFieldId()), eventField, data != null ? data.get(eventField.getCaseFieldId()) : null))
             .collect(Collectors.toList());
     }
+
+    private void getFieldTypeWithDynamicListsPopulated(CaseViewField caseField, Object value) {
+
+        if (caseField.getFieldType().getType().equals(DYNAMIC_LIST) && value != null) {
+            caseField.setValue(((ObjectNode) value).get(DEFAULT));
```

Comment:
what happens when a user selected on the ui a valued for the dynamic list and we stored that value. Would we be overriding the selected value with the default here?

Reply:
related to this, where do we read the current selected value of the dynamic list?